model.py

```diff
@@ -5698,13 +5698,13 @@ def sort_key(edition):
                 return DataSource.PRESENTATION_EDITION_PRIORITY.index(source.name)
             else:
                 return -2
+        if external_editions:
+            editions = external_editions
+        else:
+            editions = self.identifier.primarily_identifies
+        return sorted(editions, key=sort_key)
 
-        return sorted(self.identifier.primarily_identifies, key=sort_key)
-
-
-    # TODO:  policy is not used in this method.  Removing argument
-    # breaks many-many tests, and needs own branch.
-    def set_presentation_edition(self, policy=None):
+    def set_presentation_edition(self, external_editions=None):
```

Comment:
This needs a test, and a docstring explaining what's up with `external_editions`. Right now I have to guess what `external_editions` is doing. I believe it's some other list of Editions that we've decided could be components of a presentation edition, even though they're not directly connected to `self.identifier`.